internal/command/console/console.go

```diff
@@ -75,6 +85,11 @@ func runConsole(ctx context.Context) error {
 		apiClient = client.FromContext(ctx).API()
 	)
 
+	flag.ApplyAliases(ctx, map[string][]string{
+		"vm-cpus":   {"cpus"},
+		"vm-memory": {"memory"},
```

Comment:
Cobra allows to set annotations to flags, that could be where we store its aliases so flag.Get* knows  from where to fetch aliases keys.

Reply:
that would probably be *better* but it would require awkward global bookkeeping, because pflag flags don't support storing arbitrary additional data.

Reply:
It does as "Annotations" https://pkg.go.dev/github.com/spf13/pflag#FlagSet.SetAnnotation

Reply:
huh. i looked and looked but i could _not_ find that. thanks!